transformation/library/test/quadkey_test.js

```diff
@@ -0,0 +1,220 @@
+const fs = require('fs');
+const assert = require('assert').strict;
+/* Emulate how BigQuery would load the file */
+global.eval(fs.readFileSync('../../quadkey_library.js') + '');
+
+describe('QUADKEY unit tests', () => {
```

Comment:
This file containing quadkey test will block you for submitting. You can take the test directly from turf or leave it empty for now.